HCCGo/app/js/jobHistoryCtrl.js

```diff
@@ -30,8 +30,8 @@ jobHistoryModule.service('jobService', function() {
 
   }
 
-  $scope.loadJob = function(job) {
-
+  $scope.loadJob = function(job, clone) {
+    job.clone = clone;
```

Comment:
Yeah, so when they click "clone" it does the same thing as "load" except it sets that clone flag so that it knows to save it as if it were a new job. So you make your changes and submit it and then a new job pops up in jobHistory along with the original one.